src/main/java/net/sf/jabref/model/EntryTypes.java

```diff
@@ -95,8 +98,11 @@ public void removeType(String name) {
 
     }
 
-    public static final InternalEntryTypes BIBTEX = new InternalEntryTypes(BibtexEntryTypes.MISC, BibtexEntryTypes.ALL, IEEETranEntryTypes.ALL);
-    public static final InternalEntryTypes BIBLATEX = new InternalEntryTypes(BibLatexEntryTypes.MISC, BibLatexEntryTypes.ALL);
+
+    public static final InternalEntryTypes BIBTEX = new InternalEntryTypes(BibtexEntryTypes.MISC,
+            Arrays.asList(BibtexEntryTypes.ALL, IEEETranEntryTypes.ALL));
```

Comment:
Which warning does "asList" remove? I might be, that `List<EntryTypes>...` has to be changed to be changed to `EntryTypes...` Or the `...` simply be removed?